docs/source/conf.py

```diff
@@ -16,6 +16,16 @@
 
 ROOT = os.path.abspath(os.path.join("..", ".."))
 sys.path.insert(0, ROOT)
+
+import shutil  # noqa
+
+shutil.rmtree(os.path.join(ROOT, "docs/source/read-only-examples"), ignore_errors=True)
```

Comment:
I would put generated content in `docs/build` instead of `docs/source`

Reply:
I'm not sure of how this would work -- can you provide a reference?

Reply:
I did not see how these were used, so it looks like it is not only a matter of moving files around, since they need to be found by `toctree` later. I'll look into it a bit more, but there is no reason to have this blocking the PR